src/CoreBundle/DatabaseAccessLayer/DatabaseAccessLayerCmsTree.php

```diff
@@ -90,6 +96,10 @@ public function getChildren($id, $includeHidden, $languageId = null)
         /** @var \TdbCmsTree[] $matches */
         $matches = $this->findDbObjectFromFieldInCache('parent_id', $id, $languageId);
 
+        /**
+         * @psalm-suppress InvalidArgument
+         * @FIXME returning `null` from a sorting method is not allowed, should probably return `0`.
+         */
         usort($matches, function (\TdbCmsTree $a, \TdbCmsTree $b) {
```

Comment:
This sorting-closure returns `null` in an 'undecided' case - should probably be `0`